config/rbac/role.yaml

```diff
@@ -24,6 +24,23 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - ""
+  resources:
```

Comment:
I wonder if this is a rebase gone wonky - when we added service accounts, this is a duplicate section afaict